app/src/main/java/com/android/brewr/ui/explore/CoffeeInformationCard.kt

```diff
@@ -0,0 +1,104 @@
+package com.android.brewr.ui.explore
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString as buildAnnotatedString1
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.rememberAsyncImagePainter
+import coil.request.ImageRequest
+import com.android.brewr.model.coffee.Coffee
+import java.time.LocalDate
+
+@SuppressLint("DefaultLocale")
+@Composable
+fun CoffeeInformationCardScreen(coffee: Coffee, onClick: () -> Unit) {
+  Column(modifier = Modifier.fillMaxWidth()) {
+    Image(
+        painter =
+            rememberAsyncImagePainter(
+                ImageRequest.Builder(LocalContext.current)
+                    .data(coffee.imagesUrls[0])
+                    .apply(
+                        block =
+                            fun ImageRequest.Builder.() {
+                              crossfade(true)
+                            })
+                    .build()),
+        contentDescription = "Selected Image",
+        contentScale = ContentScale.Crop,
+        modifier =
+            Modifier.testTag("coffeeImage")
+                .fillMaxWidth()
+                .heightIn(min = 180.dp, max = 300.dp)
+                .clickable(onClick = onClick))
+    Text(
+        text = coffee.coffeeShopName,
+        fontSize = 20.sp,
```

Comment:
Just a small remark that the text seems too big on a small phone (for example on my android phone which is a relatively small phone). Maybe it would be better if we decrease a little bit the size !

Reply:
Fixed!! Thanks for the feedback~